clang/include/clang/Driver/Options.td

```diff
@@ -6691,12 +6692,16 @@ defm : FlangIgnoredDiagOpt<"frontend-loop-interchange">;
 defm : FlangIgnoredDiagOpt<"target-lifetime">;
 
 // C++ SYCL options
+let Group = sycl_Group in {
```

Comment:
Please take a look at few more SYCL options listed here and check if we need to clean that up. Thanks
https://github.com/mdtoguchi/llvm-project/blob/d819f8911ad6360b667289bd90eb5b6401105335/clang/include/clang/Driver/Options.td#L8204

Reply:
This section is for `-cc1` options.  The only potential cleanup is the values for `-sycl-std` but we can do that in a future patch.